src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -123,14 +124,15 @@ def _get_docker_status_and_version(ignore_errors, yes):
 
 # Get current CLI version
 def _get_cli_version():
-    acr_component_name = "azure-cli-acr"
-    acr_cli_version = "not found"
     from pkg_resources import working_set
-    for component in list(working_set):
-        if component.key == acr_component_name:
-            acr_cli_version = component.version
 
-    print('ACR CLI version: {}'.format(acr_cli_version))
+    # working_set.by_key is a dictionary with component names as key
+    cli_component_name = "azure-cli"
+    if cli_component_name in working_set.by_key:
+        print('ACR CLI version: {}'.format(
+            working_set.by_key[cli_component_name].version))
+    else:
+        print('ACR CLI version: not found', file=sys.stderr)
```

Comment:
nit: can you change this to "Azure CLI version" or something similar, since it's no longer acr cli?

Reply:
@ni-reis Yes, I have pushed those changes now.